src/IConfigOptions.ts

```diff
@@ -162,6 +162,8 @@ export interface IConfigOptions {
 
     terms_and_conditions_links?: { url: string, text: string }[];
 
+    is_tor_server?: boolean;
```

Comment:
Any options added here need to be documented in https://github.com/vector-im/element-web/blob/develop/docs/config.md. Although it's unclear to me what this option would do, it doesn't seem like the top-level config of the app is the right place to put it.